app/src/main/java/UserData.kt

```diff
@@ -0,0 +1,8 @@
+import java.io.Serializable
+
+data class UserData (
+    var id : String = "",
+    var pwd : String = "",
+    var nickname : String = "",
+    var mbti : String = ""
```

Comment:
```suggestion
    var id : String,
    var pwd : String,
    var nickname : String,
    var mbti : String
```

초기값이 필요하지 않다면 설정하지 않아도 됩니다! 요기에서는 4개 값을 모두 필수적으로 넣어주기 때문에 초기값을 사용하지 않아도 될 것 같아요 !